.ci/deps.sh

```diff
@@ -62,3 +62,24 @@ wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/ast
 tar -xvzf ~/astyle.tar.gz -C ~/
 make -C ~/astyle/build/gcc
 sudo make install -C ~/astyle/build/gcc
+
+# Elixir commands
+wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb
```

Comment:
can we add a repo instead, and let apt-get do its trick elsewhere?

If not, at least move this out to a separate `deps.mix.sh`